ReactAndroid/src/main/java/com/facebook/hermes/reactexecutor/OnLoad.cpp

```diff
@@ -30,26 +28,6 @@ static void hermesFatalHandler(const std::string &reason) {
 
 static std::once_flag flag;
 
-static ::hermes::vm::RuntimeConfig makeRuntimeConfig(jlong heapSizeMB) {
-  namespace vm = ::hermes::vm;
-  auto gcConfigBuilder =
-      vm::GCConfig::Builder()
-          .withName("RN")
-          // For the next two arguments: avoid GC before TTI by initializing the
-          // runtime to allocate directly in the old generation, but revert to
-          // normal operation when we reach the (first) TTI point.
-          .withAllocInYoung(false)
-          .withRevertToYGAtTTI(true);
```

Comment:
Note that this is a flip of [the default GCConfig](https://github.com/facebook/hermes/blob/5f2b47d0be6281fd2605d24efc0b43af42b4033d/public/hermes/Public/GCConfig.h#L185-L188) that we customized to optimize TTI for all React Native apps but not going to work on any other environments (hence not suitable to be the Hermes default).